### PR TITLE
PAPP-29406: document changes, update permissions and update summary

### PR DESCRIPTION
**Pull request by aparekh-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Review by dhwanis-crest — APPROVED** [review]

**Review by mmomin-crest — APPROVED** [review]

File: readme.html [hunk 1]
```diff
@@ -35,7 +35,130 @@ <h2>Authentication</h2>
 <p>
   SOAR's Slack App needs a bot token to read messages from and post messages to slack channels. The app also needs a
   verification token to verify POST requests received from Slack.
-<h3>Create a Slack App</h3>
+<h3>Steps for Create a Slack App Using App Manifest File (Recommended)</h3>
```
Comment (splunk-jessica) [inline readme.html:38]:
```suggestion
<h3>Steps for Creating a Slack App Using App Manifest File (Recommended)</h3>
```

File: readme.html [hunk 2]
```diff
@@ -35,7 +35,130 @@ <h2>Authentication</h2>
...
+      <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_select_workspace.png">
+</a>
+<br><br>
+Paste above manifest file in <b>yaml</b> section.
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_enter_app_manifest.png">
+      <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_enter_app_manifest.png">
+</a>
+<br><br>
+Once click on next, verify all scopes for Bot and User which is added from the manifest file.
```
Comment (splunk-jessica) [inline readme.html:113]:
```suggestion
Click <b>Next</b>, then verify all scopes for Bot and User added from the manifest file.

File: readme.html [hunk 3]
```diff
@@ -35,7 +35,130 @@ <h2>Authentication</h2>
...
+      <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_review_summary_oauth.png">
+</a>
+<br><br>
+Event subscriptions for the bot will be added from the manifest file, Click on create.
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_review_summary_settings.png">
+      <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_review_summary_settings.png">
+</a>
+<br><br>
+On the next page, there is a <b>Verification Token</b>. This token will be needed during asset configuration.
```
Comment (splunk-jessica) [inline readme.html:125]:
```suggestion
On the next page, there is a <b>Verification Token</b>. This token is needed during asset configuration.

File: readme.html [hunk 4]
```diff
@@ -35,7 +35,130 @@ <h2>Authentication</h2>
...
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_basic_info.png">
+  <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_basic_info.png">
+</a>
+<br><br>
+On the same page, there is a <b>App Level tokens</b>. This dialog box will have a <b>Generate Token and Scope</b>
```
Comment (splunk-jessica) [inline readme.html:131]:
```suggestion
On the same page, there are <b>App Level tokens</b>. This dialog box will have a <b>Generate Token and Scope</b>

File: readme.html [hunk 5]
```diff
@@ -35,7 +35,130 @@ <h2>Authentication</h2>
...
+button, click on it. It would again open a pop up, give token name <b>socket_token</b>. Just below this would be the
+<b>Add Scope</b> button, add <b>connection:write & authorization:read</b> and click on <b>Generate</b>. This token will
+be needed during asset configuration.
```
Comment (splunk-jessica) [inline readme.html:134]:
```suggestion
button. You will then be prompted to provide a token name <b>socket_token</b>. Just below this is the
<b>Add Scope</b> button. Add <b>connection:write & authorization:read</b> and click on <b>Generate</b>. This token is
needed during asset configuration.

File: readme.html [hunk 6]
```diff
@@ -35,7 +35,130 @@ <h2>Authentication</h2>
...
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_socket_token.png">
+  <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_socket_token.png">
+</a>
+<br><br>
+On the same page click <b>Install to Workspace</b> which is inside <b> Install your app</b> section.
```
Comment (splunk-jessica) [inline readme.html:140]:
```suggestion
On the same page under the <b>Install your app</b> section, click <b>Install to Workspace</b>.

File: readme.html [hunk 7]
```diff
@@ -35,7 +35,130 @@ <h2>Authentication</h2>
...
+</a>
+<br><br>
+On the next page, click <b>Allow</b>.
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_allow_app_manifest.png">
+  <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_allow_app_manifest.png">
+</a>
+<br><br>
+After authorizing the app, In the menu bar on the left, select <b>OAuth & Permissions</b>. The <b>Bot User OAuth Token
+</b> and <b>User OAuth Token</b> will be required during asset configuration.
```
Comment (splunk-jessica) [inline readme.html:153]:
```suggestion
After authorizing the app, in the menu bar on the left, select <b>OAuth & Permissions</b>. The <b>Bot User OAuth Token
</b> and <b>User OAuth Token</b> are required during asset configuration.

File: readme.html [hunk 8]
```diff
@@ -35,7 +35,130 @@ <h2>Authentication</h2>
...
+<br><br>
+<a href="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_auth_tokens.png">
+  <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_auth_tokens.png">
+</a>
+<br><br>
+On the same page, Scroll down to the <b>Scopes</b> section, verify the scopes for Bot and User which are added from the manifest file.
+<br><br>
+<h3>Steps for Create a Slack App From Scratch</h3>
```
Comment (splunk-jessica) [inline readme.html:161]:
```suggestion
On the same page, scroll down to the <b>Scopes</b> section and verify the scopes for Bot and User added from the manifest file.
<br><br>
<h3>Steps for Creating a Slack App From Scratch</h3>

File: readme.html [hunk 9]
```diff
@@ -59,7 +182,7 @@ <h3>Create a Slack App</h3>
 
 <br><br>
 On the same page, there is a <b>App Level tokens</b>. This dialog box will have a <b>Generate Token and Scope</b>
-button, click on it. It would again open an pop up, give token name <b>socket_token</b>. Just below this would be the
+button, click on it. It would again open a pop up, give token name <b>socket_token</b>. Just below this would be the
 <b>Add Scope</b> button, add <b>connection:write & authorization:read</b> and click on <b>Generate</b>. This token will
 be needed during asset configuration.
```
Comment (splunk-jessica) [inline readme.html:187]:
```suggestion
On the same page, there is an <b>App Level tokens</b> section. Click on the <b>Generate Token and Scope</b>
button. You will then be prompted to provide a token name <b>socket_token</b>. Just below this is the
<b>Add Scope</b> button. Add <b>connection:write & authorization:read</b> and click on <b>Generate</b>. This token is
needed during asset configuration.

File: readme.html [hunk 10]
```diff
@@ -206,8 +324,8 @@ <h3>Create a Slack App</h3>
   <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_allow_app.png">
 </a>
 <br><br>
-After authorizing the app, the next window will show the app's authorization tokens. The <b>Bot User OAuth Access
-  Token</b> and <b>OAuth Access Token</b> will be required during asset configuration.
+After authorizing the app, the next window will show the app's authorization tokens. The <b>Bot User OAuth Token</b>
+and <b>User OAuth Token</b> will be required during asset configuration.
```
Comment (splunk-jessica) [inline readme.html:328]:
```suggestion
and <b>User OAuth Token</b> are required during asset configuration.

File: readme.html [hunk 11]
```diff
@@ -334,6 +457,13 @@ <h2>Set up Socket Mode in Slack</h2>
       <img src="/app_resource/slack_3ac26c7f-baa4-4583-86ff-5aac82778a86/img/slack_subscription_events.png">
     </a>
     <br><br>
+    Note: When socket mode is enabled, On left menu bar click on <b>Interactivity & Shortcuts</b> and verify that <b>Request URL</b> field should be empty in <b>interactivity</b> section and
+    It should be looked like below.
```
Comment (splunk-jessica) [inline readme.html:461]:
```suggestion
    Note: When socket mode is enabled, click on <b>Interactivity & Shortcuts</b> on the left menu bar and verify that the <b>Request URL</b> field is empty in the <b>Interactivity</b> section.

File: readme.html (hunk 5 above)
Comment (splunk-jessica) [reply readme.html:134]: @aparekh-crest does the token name entered on for this prompt have to be "socket_token"?

Comment (splunk-jessica) [reply readme.html]: @aparekh-crest ok good to know. Then I would suggest revising the sections with references to "socket_token" to something like this

> button. You will then be prompted to provide a token name, for example, <b>socket_token</b>. Just below this is the

Comment (aparekh-crest) [reply readme.html]: @splunk-jessica Sounds good, will do that.

**Review by splunk-jessica — APPROVED** [review]

**Pull request merged**